sdk/core/azure-core-http-netty/CHANGELOG.md

```diff
@@ -1,7 +1,15 @@
 # Release History
 
-## 1.11.0-beta.1 (Unreleased)
+## 1.11.0 (Unreleased)
 
+### Features Added
+
+- Added the ability to configure HTTP connect timeout. ([#23435](https://github.com/Azure/azure-sdk-for-java/pull/23435))
+- Added support for additional environment configurations. ([#23435](https://github.com/Azure/azure-sdk-for-java/pull/23435))
+
+### Dependency Updates
```

Comment:
Following latest changelog guidelines, 
we can move this section under `Other Changes` header.
Here and other places too in the PR.

### Other Changes
#### Dependency Updates